ssh/ssh_test.go

```diff
@@ -515,7 +515,7 @@ func TestSSHGetExeAndArgsInvalidOptionsAsPath(t *testing.T) {
 	assert.Equal(t, "git@git-host.com", e.SSHMetadata.UserAndHost)
 	assert.Equal(t, "/-oProxyCommand=gnome-calculator", e.SSHMetadata.Path)
 
-	exe, args, needShell := ssh.GetExeAndArgs(cli.OSEnv(), cli.GitEnv(), &e.SSHMetadata, false)
+	exe, args, needShell, _ := ssh.GetExeAndArgs(cli.OSEnv(), cli.GitEnv(), &e.SSHMetadata, false)
```

Comment:
All of the existing tests just ignore the new return value ... should we add some tests which confirm its true/false state in particular conditions?

Reply:
I can look into that.  Part of this depends on whether we can create a specific temporary directory, which is potentially tricky and I was concerned might lead to flaky tests, but I'll see what I can do to try to make that less of a problem.

Reply:
Thanks for the extra test; looks good!